data/json/recipes/recipe_weapon.json

```diff
@@ -4633,11 +4633,11 @@
   "time": 8000,
   "reversible": true,
   "autolearn": true,
+  "using": [ [ "adhesive", 1 ] ],
   "tools": [],
   "components": [
     [ [ "rock", 5 ], [ "ceramic_shard", 5 ], [ "sharp_rock", 5 ] ],
-    [ [ "stick", 1 ], [ "2x4", 1 ] ],
-    [ [ "superglue", 1 ], [ "bone_glue", 2 ] ]
```

Comment:
It's easier to read if you keep all the components under components. Instead of "using" use "adhesive", 1, "LIST" like  you did in the other files.

Reply:
Most recipes prefer to use `using` when possible, unless they've gone and changed the preferred procedure. Again.

Reply:
@smolbird I think that's more for requirements with tools and components or tool sets in general. Going forward I think keeping all the components together when possible is preferred and easier to read.

Reply:
Hmm. Both are actually in use in the files, so I'm not sure which is preferred by the devs at present.